### PR TITLE
feat(website): Blog for Privilege Escalation with AWS Identity Center

### PR DESCRIPTION
**Pull request by jsonpr**

### CODE REVIEW

File: website/pages/blog/aws-priv-esc-identity-center.mdx [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+
+<BlogHeader/>
+
+## Overview
+
+AWS Identity Center ([formerly known as AWS SSO](https://docs.aws.amazon.com/singlesignon/latest/userguide/what-is.html#renamed)) is one method of providing and managing cloud access to Amazon Web Services accounts within an Organization, resources in AWS accounts, and applications hosted in AWS.  As with other methods of access to AWS accounts including IAM Roles and Users, there is the potential for misconfiguration and paths to privilege escalation and other security issues within AWS environments.
+
+In this blog, we will cover methods for privilege escalation with AWS Identity Center, access disruption, and other events that could indicate security issues within an AWS environment.  We’ll cover our research and observations on AWS Identity Center from our deep dive and experimentation with AWS Identity Center, with a specific focus on AWS Identity Center and attack vectors introduced by the usage of AWS Identity Center.  There are existing methods of privilege escalation that have been covered and researched by the security community such as `iam:PassRole` , `sts:AssumeRole` , and `iam:SetDefaultPolicyVersion`.
```
Comment (yevgenypats) [inline website/pages/blog/aws-priv-esc-identity-center.mdx:18]: Maybe a link here to previous research (for `iam:PassRole`, `sts:AssumeRole` and `SetDefaultPolicy`)?

File: website/pages/blog/aws-priv-esc-identity-center.mdx [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+
+For example, to add an inline policy, both `PutPermissionPolicy` and `PutInlinePolicyToPermissionSet` could work.
+
+![Dual Authorization Table Provided by AWS](/images/blog/aws-priv-esc-identity-center/dualauth.png)
+
+Dual Authorization Table Provided by AWS
+
+## Privilege Escalation with AWS Identity Center (SSO)
+
+Identity Center has multiple service prefixes including:
```
Comment (yevgenypats) [inline website/pages/blog/aws-priv-esc-identity-center.mdx:73]: I think I miss a piece here or a clarification is needed. are those permissions that if they are granted the user can add himself permissions and/or himself to groups? If yes, better to state that then explicitly.

**Review by yevgenypats — CHANGES_REQUESTED** [review]
Looks good. Did a first round of review.

File: website/pages/blog/aws-priv-esc-identity-center.mdx (hunk 1 above)
Comment (jsoncow) [reply website/pages/blog/aws-priv-esc-identity-center.mdx]: Good point, added!

Comment (jsonpr) [reply website/pages/blog/aws-priv-esc-identity-center.mdx]: Good point!  Added.

File: website/pages/blog/aws-priv-esc-identity-center.mdx (hunk 2 above)
Comment (jsonpr) [reply website/pages/blog/aws-priv-esc-identity-center.mdx:75]: Added!

**Review by yevgenypats — APPROVED** [review]

**Pull request merged**